aries_cloudagent/protocols/didexchange/v1_0/manager.py

```diff
@@ -822,6 +843,13 @@ async def accept_response(
             await self.record_did(response.did)
 
         conn_rec.their_did = their_did
+
+        # The long format I sent has been acknoledged, use short form now.
+        if LONG_PATTERN.match(conn_rec.my_did or ""):
+            conn_rec.my_did = await self.long_did_peer_4_to_short(conn_rec.my_did)
```

Comment:
I think as a community we need to formalize this a bit more but I think this is a logical starting point.